nimbusCBT/client/src/App.jsx

```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+import EmailValidator from "./components/EmailValidation.jsx";
+
+function App() {
+  const [count, setCount] = useState(0);
+
+  return (
+    <div className="App">
+      <h1>Email thing</h1>
```

Comment:
Is this PR going to become the module we later use in the app? I.e. are we just going to reuse this code when the app is more fleshed out and we need to incorporate this feature? 

If so, then maybe we should make some of the boilerplate more official looking/parameterized so it'll be easier to implement in the future. We can discuss Sunday depending your answer